src/apiCalls/axiosInstance.ts

```diff
@@ -0,0 +1,96 @@
+import { buildAxiosFetch } from '@lifeomic/axios-fetch';
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+const fetch = buildAxiosFetch(axios);
+
+const getFormattedAxiosResponse = async <T>(response: Response, config?: T) => {
```

Comment:
add explenation here:
// Needs to be used beacause an async call made after cross-window user interaction makes the dapp unresponsive